server/src/index.js

```diff
@@ -11,7 +11,7 @@ const routes = require("./routes.js");
 // const autodis = require("./discord/autodis.js");
 // const bpl3 = require("./bpl3.js");
 
-let domains = ["slmn.gg", "localslmn", "localhost"].map(d => new RegExp(`(?:^|.*\\.)${d.replace(".", "\\.")}(?:$|\\n)`));
+let domains = process.env.CORS_VALID_DOMAINS.split(", ").map(d => new RegExp(`(?:^|.*\\.)${d.replace(".", "\\.")}(?:$|\\n)`));
```

Comment:
```suggestion
let domains = (process.env.CORS_VALID_DOMAINS || "slmn.gg").split(/, */g).map(d => new RegExp(`(?:^|.*\\.)${d.replace(".", "\\.")}(?:$|\\n)`));
```
More fallbacks, letting the test use `localhost, slmn.gg` and `localhost,slmn.gg`; also another fallback